tools/ossf_best_practices/silver_criteria.md

```diff
@@ -91,7 +92,7 @@ The project MUST make an effort to keep the documentation consistent with the cu
 
 The project repository front page and/or website MUST identify and hyperlink to any achievements, including this best practices badge, within 48 hours of public recognition that the achievement has been attained. (URL required) 
 
-**Met**
+**Met** https://github.com/nodejs/nodejs.org/issues/5432
```

Comment:
This is a temporal reference, until we manage to have the badge in the website 🤔